Document/0x06b-Basic-Security-Testing.md

```diff
@@ -401,10 +435,46 @@ PID  Name
 499  Gadget
 ~~~
 
+![Frida on non-JB device](/Document/Images/Chapters/0x06b/fridaStockiOS.png "Frida on non-JB device")
+
 ##### Troubleshooting.
 
 If something goes wrong (which it usually does), mismatches between the provisioning profile and code signing header are the most likely suspect. In that case it is helpful to read the official documentation and gaining an understanding of how the whole system works [7][8]. I also found Apple's entitlement troubleshooting page [9] to be a useful resource.
 
+
+### Setting up Burp
+Setting up burp to proxy your traffic through is pretty straightforward. It is assumed that you have both: iDevice and workstation connected to the same WiFi network where client to client traffic is permitted. If client-to-client traffic is not permitted, it should be possible to use usbmuxd [18] in order to connect to burp through USB. 
+
+The first step is to configure proxy of your burp to listen on all interfaces (alternatively only on the WiFi interface), as per screenshot.
+
+![Setting up Burp Proxy](/Document/Images/Chapters/0x06b/setBurpProxy.png "Setting up Burp Proxy")
+
+Then we can configure our iDevice to use our proxy in advanced wifi settings. 
```

Comment:
Can you please just reference to the Portswigger documentation and shorten this paragrpah? It is already explained there in a good way https://support.portswigger.net/customer/portal/articles/1841108-configuring-an-ios-device-to-work-with-burp